crates/ruff_linter/src/rules/flake8_logging_format/rules/logging_call.rs

```diff
@@ -153,22 +153,36 @@ impl LoggingCallType {
 
 /// Check logging calls for violations.
 pub(crate) fn logging_call(checker: &mut Checker, call: &ast::ExprCall) {
-    let Expr::Attribute(ast::ExprAttribute { value: _, attr, .. }) = call.func.as_ref() else {
-        return;
-    };
-
-    let Some(logging_call_type) = LoggingCallType::from_attribute(attr.as_str()) else {
-        return;
+    // Determine the call type (e.g., `info` vs. `exception`) and the range of the attribute.
+    let (logging_call_type, range) = match call.func.as_ref() {
+        Expr::Attribute(ast::ExprAttribute { value: _, attr, .. }) => {
+            let Some(call_type) = LoggingCallType::from_attribute(attr.as_str()) else {
+                return;
+            };
+            if !logging::is_logger_candidate(
+                &call.func,
+                checker.semantic(),
+                &checker.settings.logger_objects,
+            ) {
+                return;
+            }
+            (call_type, attr.range())
+        }
+        Expr::Name(_) => {
+            let Some(call_path) = checker.semantic().resolve_call_path(call.func.as_ref()) else {
```

Comment:
@qdegraaf - I changed this to always use `resolve_call_path`. I think we want this to _only_ match the logging imports.